sentry/src/main/java/io/sentry/ISentryClient.java

```diff
@@ -95,52 +95,6 @@ default SentryId captureMessage(String message, SentryLevel level) {
     return captureMessage(message, level, null);
   }
 
-  /**
```

Comment:
Why are we getting rid of this API?

Reply:
looks like wrong

Reply:
When exception is captured there is a lookup in Hub to assign a trace context. If we allow SentryEvents to be constructed in the `SentryClient` they will never have trace context attached. 

I don't see the reason why should we keep these methods in `ISentryClient` but maybe I am missing something?

Reply:
Java 1.7.x didn't have the Static class but only "SentryClient" and they were using it directly, I guess this could break a lot of user's code (if people migrated the easy way).
Also, people developing libraries would not use the Static class, either Hub or SentryClient, that would be another segment that would be broken.
I don't think removing them would be a good thing, but rather documenting the limitation.
the other possibility would be to `assignTraceContext` in the SentryClient, but comes with the probably of whom should hold the state, Hub or SentryClient? in theory, only the Hub

Reply:
Yes it's about the state - that has been already discussed. I'm ok to bring back these methods. Maybe I misunderstood @bruno-garcia when we discussed removing them as they are a part of interface that's not used directly by users anyway.

Reply:
Oh I think there was a misunderstanding.
Instead of removing this, lets just document that for connecting errors to span, `Hub` must be involved since that's what adds state to Sentry. Accessing the Client doesn't add any Scope or other state so it shouldn't be a surprise

Reply:
Also, can't we inspect `event.getThrowable()` when one calls `captureEvent`?

Reply:
Methods are back now.